<launch>
  <node pkg="multirobot_detect_iarc" name="multirobot_detect_iarc_node" type="multirobot_detect_iarc_node" output="screen">
  <!--param name="subscribed_topic" type="string" value="/dji_sdk/image_raw"/-->
  <param name="subscribed_topic" type="string" value="/my_video"/>
  <param name="show_video_flag" type="bool" value="true"/>
  <param name="save_result_video_flag" type="bool" value="true"/>
  <param name="result_video_file_name" type="string" value="/home/zj/ros_workspace/src/multirobot_detect_iarc/result/raw.avi"/>
  <param name="rate" type="double" value="5.0"/>
  <param name="save_set_flag" type="bool" value="false"/>
  </node>
</launch>
